processor/t60_processor.py

```diff
@@ -26,75 +31,158 @@ def extract_s3_info(object_url: str) -> [str, str]:
         self._S3_READ_CLIENT = client(
             "s3",
-            region_name=environ.get('AWS_BUCKET_REGION')
+            region_name=environ.get('AWS_BUCKET_REGION'),
+            aws_access_key_id=environ.get('AWS_ACCESS_KEY_ID'),
+            aws_secret_access_key=environ.get('AWS_SECRET_ACCESS_KEY')
         )
         self.skip_frame = environ.get('SKIP_FRAME', 3)
-        self.video_path = environ.get('VIDEO_PATH')
-        self.camera_path = environ.get('CAMERA_PATH')
-        self.output_path = ''
+        self.video_path = environ.get('VIDEO_PATH', 'https://s3.amazonaws.com/ai.powern.website.assets/cpms/FWH/360Slam/video.mp4')
+        self.camera_path = environ.get('CAMERA_PATH','https://s3.amazonaws.com/ai.powern.website.assets/cpms/FWH/360Slam/equirectangular.yaml')
+        self.output_path = 'messagepack.msg'
+        self.bucket = ''
+        self.parent_key = ''
+        self.dest_path = './360Images/'
+        self.coordinates_file = 'coordinates.json'
 
     def localize(self):
         '''
             Run the stella-vslam localizer to extract the map information
+            --auto-term end is required to complete process in the background
         '''
         subprocess.run([
-            './run_video_slam', '-v', '/tmp/data/orb_vocab.fbow', '-m', self.video_path, '-c', self.camera_path, '--frame-skip', self.skip_frame, '--no-sleep',
-            '--map-db-in', self.output_path
-        ])
+                './run_video_slam', '-v', './orb_vocab.fbow', '-m', self.video_local_path, '-c', self.camera_local_path, '--frame-skip', str(self.skip_frame), '--no-sleep',
+                '--map-db-out', self.output_path, '--auto-term'
+            ])
+    def extract_frames_and_coordinates(self):
+        '''
+            Extracts the msgpack from the output path
+        '''
+        service = FrameCoordinatesExtractor(self.output_path, self.video_local_path, self.dest_path) 
+        service.generate_frames_from_msg()
+        service.generate_final_coordinates_json()
+        
+        
+    def set_bucket_and_parent_key(self):
+        '''
+            Sets the object and parent key for the output path
+        '''
+        self.bucket, object_key = Utils.extract_s3_info(self.video_path)
+        self.parent_key = Utils.extract_parent_key(object_key)
+
 
     def download(self, source_url: str):
         """
-            Downloads the file object from Source (s3) and creates a temporary file
+        Downloads the file object from Source (s3) and creates a temporary file
         """
-        bucket, object_key = Utils.extract_s3_info(
-            source_url.get("s3_path"))
-
+        if isinstance(source_url, dict):
+            bucket, object_key = Utils.extract_s3_info(source_url.get("s3_path"))
+        else:
+            # Handle the case when source_url is a string
+            bucket, object_key = Utils.extract_s3_info(source_url)
         temp_file = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
we have been running into issues where the file sizes have been greater than tmpfs, can you look at the alternative where the tmp path can be given the EBS Volume path instead